de.fhdo.lemma.technology.technologydsl.extractor/src/de/fhdo/lemma/technology/technologydsl/extractor/Util.xtend

```diff
@@ -0,0 +1,87 @@
+package de.fhdo.lemma.technology.technologydsl.extractor
+
+import org.eclipse.emf.ecore.EObject
+import de.fhdo.lemma.data.Context
+import de.fhdo.lemma.data.ComplexType
+import de.fhdo.lemma.data.DataOperation
+import de.fhdo.lemma.data.DataOperationParameter
+import de.fhdo.lemma.data.DataField
+import de.fhdo.lemma.data.EnumerationField
+
+/**
+ * This class collects _static_ utility methods for the Data DSL extractor.
```

Comment:
Err? Please adapt the class and its methods to comply with the Technology DSL extractor.

Reply:
Is the class actually required by the extractor because most of its methods are directly aligned to the Data DSL?!